### ISSUE TITLE
Cannot access p5's Vector

### ISSUE THREAD

**Issue by finnbear** [tonyketcham/p5-svelte#24]
**Describe the bug**
I am unable to access p5's native `Vector` (https://p5js.org/reference/#/p5.Vector), for the purpose of calling its static methods such as `p5.Vector.fromAngle(angle)`

Issue probably applies to all p5's native classes.

**To Reproduce**
1. Create sveltekit app (may also apply to sapper)
2. Follow the setup suggested here https://github.com/tonyketcham/p5-svelte/issues/6#issuecomment-771032511
3. Attempt to access `p5.Vector` or `P5.Vector`
4. Both result in `undefined`

**Expected behavior**
Rather than `undefined`, `p5.Vector` should be p5's native `Vector`

**Comment by tonyketcham** [comment]
Just letting you know that I've got a fix for this coming in the next release! 

Right now I'm considering adding non-breaking support where these classes can be accessed by passing additional optional parameters to the `sketch` function:
```js
const sketch = (p5, Vector) => {
  p5.draw = () => {
    let v = Vector.fromAngle(p5.radians(120), 30);
    console.log(v) // --> l.default.Vector {x: -14.999999999999993, y: 25.98076211353316, z: 0}
  };
};
```

Since these will be optional, the original syntax also works:
```js
const sketch = (p5) => {
  let x = 100;
  let y = 100;

  p5.draw = () => {
    p5.background(0);
    p5.fill(255);
    p5.rect(x, y, 50, 50);
  };
};
```

Do you think this is a good approach, @finnbear?

**Comment by tonyketcham** [comment]
I've improved the API to abide by standard p5 instance mode syntax, where the following will be possible and you won't have to worry about passing in any additional props:
```js
const sketch = (p5) => {
  p5.draw = () => {
    let v = p5.Vector.fromAngle(p5.radians(120), 30);
    console.log(v) // --> l.default.Vector {x: -14.999999999999993, y: 25.98076211353316, z: 0}
  };
};
```

**Comment by finnbear** [comment]
Hi there, that second API looks good to me :+1:  (don't take my opinion too seriously though, as I migrated to another graphics framework).

**Comment by tonyketcham** [comment]
Resolved in [`v3.0`](https://github.com/tonyketcham/p5-svelte/releases/tag/v3.0.0) 🎉
